tokio-postgres/src/client.rs

```diff
@@ -313,6 +313,36 @@ impl Client {
     /// Panics if the number of parameters provided does not match the number expected.
     ///
     /// [`query`]: #method.query
+    ///
+    /// # Examples
+    ///
+    /// If you have a type like `Vec<T>` where `T: ToSql` Rust will not know how to use it as params. To get around
+    /// this the type must explicitly be converted to `&dyn ToSql`.
+    ///
+    /// ```no_run
+    /// # { #![cfg(feature = "runtime")] // tokio_postgres::connect doesn't exist without "runtime"
```

Comment:
There was some trickery here to make it work with `cargo test --no-default-features --doc`.

Reply:
Instead of that, you can change the signature of `async_main` to pass the `Client` in and avoid having to cfg at all.

Reply:
Great suggestion, way less clutter now.